custom/icds_reports/templates/icds_reports/icds_app/mobile/map-chart.html

```diff
@@ -28,7 +28,7 @@
       </div>
     </div>
     <div ng-if="$ctrl.step === 'chart'" style="padding-top: 10px">
-      <div class="chartView col-xs-11">
+      <div class="chartView col-xs-11" ng-if="!$ctrl.hideRanking" >
```

Comment:
Ported from the web version. I think one or two pages don't show this and I didn't want to forget. Though I will test it properly when I port those pages across.

Reply:
i think on web dashboard its map page which does not show.

Reply:
sorry, what do you mean?